File: Server/Models/Answer.cs
using System.ComponentModel.DataAnnotations;
using WebApiQandA.Models;

namespace Server2.Models
{
	public class Answer//ответ
	{
		public int Id { set; get; }//ид ответа

		[StringLength(100)]
		public string TextAnswer { set; get; }//текст ответа

		public int IdSurvey { set; get; }//ид опроса, чтобы легче найти было

		public Survey Survey { set; get; }//ссылка на опрос

		public Vote[] Votes { set; get; }
	}
}

File: QandAProject/src/app/classes/answer.ts
import { Vote } from './vote';

export class Answer {

  public id: number;

  public idSurvey: number;

  public textAnswer: string;

  public votes: Vote[];

  public isUserVote: boolean;

  constructor(idSurvey: number, textAnswer: string) {
    this.id = 0;
    this.idSurvey = idSurvey;
    this.isUserVote = false;
    this.textAnswer = textAnswer;
    this.votes = new Array<Vote>();
  }

}
